ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Extension.java

```diff
@@ -59,6 +59,7 @@
     public static final String NAMESPACE_2_0 = EJB3SubsystemNamespace.EJB3_2_0.getUriString();
     public static final String NAMESPACE_3_0 = EJB3SubsystemNamespace.EJB3_3_0.getUriString();
     public static final String NAMESPACE_3_1 = EJB3SubsystemNamespace.EJB3_3_1.getUriString();
+    public static final String NAMESPACE_4_0 = EJB3SubsystemNamespace.EJB3_4_0.getUriString();
 
```

Comment:
3.1 was just introduced in WF10 - so I don't think you need to increment the version.